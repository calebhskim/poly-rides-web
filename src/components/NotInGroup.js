import React from 'react';

import Paper from 'material-ui/Paper';

import cardStyle from '../styles/components/card';
import styles from '../styles/components/general';

const msg = 'Oh no! It looks like you\'re not in the Cal Poly Ride Share Group. Request to join ';
const NotInGroup = () => (
  <Paper style={Object.assign({}, cardStyle, styles.container)} >
    <h4>
      {msg}
      <a
        href='https://www.facebook.com/groups/250502971675365/'
        target='RIDESHAREGROUP'
      >
        here
      </a>.
    </h4>
  </Paper>
);

export default NotInGroup;
