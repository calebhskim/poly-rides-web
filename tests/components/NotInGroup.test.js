import React from 'react';

import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import NotInGroup from '../../src/components/NotInGroup';

describe('<NotInGroup />', () => {
  it('renders properly', () => {
    const wrapper = shallow(<NotInGroup />);

    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
