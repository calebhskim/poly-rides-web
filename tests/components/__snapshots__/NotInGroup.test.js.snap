// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotInGroup /> renders properly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
    }
  }
>
  <h4>
    Oh no! It looks like you're not in the Cal Poly Ride Share Group. Request to join 
    <a
      href="https://www.facebook.com/groups/250502971675365/"
      target="RIDESHAREGROUP"
    >
      here
    </a>
    .
  </h4>
</div>
`;
